hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -197,7 +241,7 @@ def publish_files(self, task_settings, item, publish_path):
         alembic_args.append("-file %s" % publish_path.replace("\\", "/"))
 
         # Set the root node to be exported
-        alembic_args.append("-root %s" % item.get_property("lod_full_name"))
+        alembic_args.append("-root %s" % item.properties.fields["node"])
```

Comment:
Please don't revert this back to `node`, `lod_full_name` was added to allow same names in the hierarchy for WIP publishes.

Reply:
Addressed